deployments/ansible/molecule/with_instrumentation/verify.yml

```diff
@@ -16,10 +16,6 @@
     - debug:
         var: ansible_facts.packages['splunk-otel-auto-instrumentation'][0]
 
-    - name: Assert specified version of splunk-otel-auto-instrumentation is installed
```

Comment:
Technically unrelated to the rest of this PR, but required for tests to pass. As a part of https://github.com/signalfx/splunk-otel-collector/pull/6284 we moved from using a preset version of `splunk-otel-auto-instrumentation` to `latest`. This check was succeeding because `latest` == `0.126.0`, but now that `0.127.0` is released, it's failing. Also, here's the [context and discussion](https://github.com/signalfx/splunk-otel-collector/pull/6284#discussion_r2127385784) for originally moving from a set version to `latest`.